docker/Dockerfile

```diff
@@ -12,4 +12,4 @@ COPY ./service .
 # USER cleanapper
 
 EXPOSE 8080/tcp
-CMD "./service" "--mysql_password=${MYSQL_APP_PASSWORD}" "--mysql_host=cleanappdb" # shell form to get env substitution.
```

Comment:
Any reason for discrepancy in names?

Reply:
No critical reason, just a readability. Having a delimiter between project name and service name.